package.json

```diff
@@ -48,11 +48,74 @@
         "command": "tacosontitan.toolbox.azure.devops.createDefaultTasks",
         "title": "Hazel's Toolbox: Create Default Azure DevOps Tasks"
       },
+      {
+        "command": "tacosontitan.toolbox.azure.devops.setWorkItem",
+        "title": "Set Work Item",
```

Comment:
Command titles should follow the convention `Hazel's Toolbox: <COMMAND_DISPLAY_NAME>`.